packages/cmr-client/tests/EarthdataLogin/test-EarthdataLoginToken.js

```diff
@@ -0,0 +1,345 @@
+'use strict';
```

Comment:
There is only one test for EarthdataLogin, and I feel that there is no need to have the EarthdataLogin directory unless there are multiple tests

Reply:
The helper functions go to the test file test-EarthdataLogin.js